sdk/daml-lf/interpreter/src/main/scala/com/daml/lf/speedy/SBuiltinFun.scala

```diff
@@ -2212,27 +2168,20 @@ private[lf] object SBuiltinFun {
                     // In Validation mode, we always call validateContractInfo
                     // In Submission mode, we only call validateContractInfo when src != dest
                     val needValidationCall: Boolean =
-                      if (machine.validating) {
-                        upgradingIsEnabled
-                      } else {
-                        // we already check qualified names match
-                        upgradingIsEnabled && (srcTmplId.packageId != dstTmplId.packageId)
-                      }
+                      machine.validating || srcTmplId.packageId != dstTmplId.packageId
```

Comment:
Looking at the diff, we used to test for `upgradingIsEnabled` here and now we don't. Is it because `upgradingIsEnabled` is now obsolete and upgrades are always on? Or is it because the check for `upgradingIsEnabled` is now done somewhere else?